### PR TITLE
Update running-a-full-node.mdx

### PR DESCRIPTION
**Pull request by jiOnederfull**
I add the information about `Beacon chain RPC connection`.

### CODE REVIEW

File: arbitrum-docs/node-running/how-tos/running-a-full-node.mdx [hunk 1]
```diff
@@ -45,7 +45,8 @@ Even though there are alpha and beta versions of the <a data-quicklook-from='arb
 ### Required parameters
 
 - L1 RPC URL
-  - Use the parameter `--parent-chain.connection.url=<Layer 1 Ethereum RPC URL>`
+  - Use the parameter `--parent-chain.connection.url=<Layer 1 Ethereum RPC URL>` for execution layer
+  - Use the parameter `--parent-chain.blob-client.beacon-url=<Layer 1 Ethereum Beacon RPC URL>` for consensus layer
```
Comment (TucksonDev) [inline arbitrum-docs/node-running/how-tos/running-a-full-node.mdx:49]:
```suggestion
  - If the chain is running [ArbOS 20](/node-running/reference/arbos-software-releases/arbos20.mdx), additionally use the parameter `--parent-chain.blob-client.beacon-url=<Layer 1 Ethereum Beacon RPC URL>` for consensus layer. You can find a list of beacon chain RPC providers [here](/node-running/reference/ethereum-beacon-rpc-providers.mdx)
```

Thanks! Can you add this extra context?

File: arbitrum-docs/node-running/how-tos/running-a-full-node.mdx [hunk 2]
```diff
@@ -45,7 +45,8 @@ Even though there are alpha and beta versions of the <a data-quicklook-from='arb
 ### Required parameters
 
 - L1 RPC URL
-  - Use the parameter `--parent-chain.connection.url=<Layer 1 Ethereum RPC URL>`
+  - Use the parameter `--parent-chain.connection.url=<Layer 1 Ethereum RPC URL>` for execution layer
+  - If the chain is running [ArbOS 20](/node-running/reference/arbos-software-releases/arbos20.mdx), additionally use the parameter `--parent-chain.blob-client.beacon-url=<Layer 1 Ethereum Beacon RPC URL>` for consensus layer. You can find a list of beacon chain RPC providers [here](/node-running/reference/ethereum-beacon-rpc-providers.mdx)
```
Comment (symbolpunk) [inline arbitrum-docs/node-running/how-tos/running-a-full-node.mdx:49]:
```suggestion
  - If the chain is running [ArbOS 20](/node-running/reference/arbos-software-releases/arbos20.mdx), additionally use the parameter `--parent-chain.blob-client.beacon-url=<Layer 1 Ethereum Beacon RPC URL>` for consensus layer. You can find a list of beacon chain RPC providers [here](/node-running/reference/ethereum-beacon-rpc-providers.mdx).
```

File: arbitrum-docs/node-running/how-tos/running-a-full-node.mdx [hunk 3]
```diff
@@ -45,7 +45,8 @@ Even though there are alpha and beta versions of the <a data-quicklook-from='arb
 ### Required parameters
 
 - L1 RPC URL
-  - Use the parameter `--parent-chain.connection.url=<Layer 1 Ethereum RPC URL>`
+  - Use the parameter `--parent-chain.connection.url=<Layer 1 Ethereum RPC URL>` for execution layer
```
Comment (symbolpunk) [inline arbitrum-docs/node-running/how-tos/running-a-full-node.mdx:48]:
```suggestion
  - Use the parameter `--parent-chain.connection.url=<Layer 1 Ethereum RPC URL>` for execution layer.
```

File: arbitrum-docs/node-running/how-tos/running-a-full-node.mdx [hunk 4]
```diff
@@ -45,7 +45,8 @@ Even though there are alpha and beta versions of the <a data-quicklook-from='arb
...
+  - If the chain is running [ArbOS 20](/node-running/reference/arbos-software-releases/arbos20.mdx), additionally use the parameter `--parent-chain.blob-client.beacon-url=<Layer 1 Ethereum Beacon RPC URL>` for consensus layer. You can find a list of beacon chain RPC providers [here](/node-running/reference/ethereum-beacon-rpc-providers.mdx)
   - It must provide a standard layer 1 node RPC endpoint that you run yourself or from a node provider
```
Comment (symbolpunk) [inline arbitrum-docs/node-running/how-tos/running-a-full-node.mdx:50]:
```suggestion
  - It must provide a standard layer 1 node RPC endpoint that you run yourself or from a node provider.
```

File: arbitrum-docs/node-running/how-tos/running-a-full-node.mdx [hunk 5]
```diff
@@ -45,7 +45,8 @@ Even though there are alpha and beta versions of the <a data-quicklook-from='arb
...
   - Note: this parameter was called --l1.url in versions prior to `v2.1.0`
```
Comment (symbolpunk) [inline arbitrum-docs/node-running/how-tos/running-a-full-node.mdx:51]:
```suggestion
  - Note: this parameter was called `--l1.url` in versions prior to `v2.1.0`
```

**Review by symbolpunk — APPROVED** [review]
Thank you!

**Pull request merged**